spoutbreeze-manager/src/main/java/org/spoutbreeze/spoutbreezemanager/SpoutbreezeManagerApplication.java

```diff
@@ -26,13 +26,14 @@
 import org.springframework.boot.ExitCodeGenerator;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
 @EnableScheduling
-@ComponentScan({ "org.spoutbreeze.spoutbreezemanager.services", "org.spoutbreeze.spoutbreezemanager.queue" })
+@EntityScan("org.spoutbreeze.commons.*")
```

Comment:
Our entity classes lie outside the scope of the current spring session. Hence, we have to tell it to get the beans from the given package. (which is there as a jar inside the current spring session)